lib/scrape.js

```diff
@@ -184,6 +185,33 @@ async function findOrganization({
   return null
 }
 
+async function fetchSEtag(wikidata_id) {
+  const url = wdk.getEntities({
+    ids: [wikidata_id],
+    props: ['claims'],
+  })
+  const page = await fetch(url)
+  let data = await page.json()
```

Comment:
Why use `await` for `page.json()`?

Reply:
Because it returns a promise?
https://developer.mozilla.org/en-US/docs/Web/API/Body/json

Reply:
Well, it just weird to use await twice for grabbing one data you need. Anyway, have you read this?

> The Promise returned from fetch() won’t reject on HTTP error status even if the response is an HTTP 404 or 500. Instead, it will resolve normally (with ok status set to false), and it will only reject on network failure or if anything prevented the request from completing.

Well, WikiData rarely encounter 500 error.

Reply:
Wikipedia API suggests that the encountering is negligible? And even if it does encounter that won't we just end up with a null object then? I don't think we have handled for that edge case in the other parsings as well..

Reply:
@wisn I did use `node-fetch` when I started this project which has native Fetch-like api thus needing that 2 await.

Bad choice really, we should make a task for switching to something else, e.g. [axios](https://github.com/axios/axios)

Reply:
@blazeu I see. Yes, we need something else. I up vote axios.

Reply:
@nalinbhardwaj

> And even if it does encounter that won't we just end up with a null object then

Yes, but, a null object doesn't have `entities` attribute. I just thinking about possibility.